packages/react-button/src/components/Button/Button.test.tsx

```diff
@@ -1,9 +1,18 @@
 import * as React from 'react';
 import * as path from 'path';
 import { isConformant } from '@fluentui/react-conformance';
-import { Button } from './Button';
+import { Button as ButtonComponent } from './Button';
 import * as renderer from 'react-test-renderer';
 import { mount, ReactWrapper } from 'enzyme';
+import { MergeStylesProvider } from '@fluentui/react-theme-provider';
+import { ButtonProps } from './Button.types';
+
+/** Use merge-styles provider to ensure styles show up in snapshots. */
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>((props, ref) => (
```

Comment:
@xugao This was kind of annoying but it's the only way to ensure styles are rendered. Do you have any other ideas here to make this easier?

Reply:
i can't think anything better than this: https://stackoverflow.com/questions/59017016/is-it-possible-to-wrap-each-individual-test-in-a-wrapper-component